.travis.yml

```diff
@@ -3,8 +3,6 @@ before_script:
   - git config --global user.name "Your Name"
 script: bundle exec rspec
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - rbx
+  - 2.2.3
+  - 2.4.0
```

Comment:
Looks like major to me since it's no longer compatible with some users. Why are they dropped?